src/app.js

```diff
@@ -1 +1,30 @@
 'use strict';
+
+const getSum = (nums) => {
+  return nums.reduce((previous, current) => previous + current);
+};
+
+const getMultiplied = (nums) => {
+  return nums.reduce((previous, current) => previous * current);
+};
+
+function getResult(enteredData) {
+  const operation = enteredData[0];
+  const numbers = enteredData.slice(1).filter(item => !isNaN(+item));
```

Comment:
your items still strings and when you sum them using 'getSum' them concatenation occurs
check: node src/app.js add 1 2 3 4 5